docs/ref-gitrepo.md

```diff
@@ -109,4 +109,13 @@ spec:
   # If empty, the "default" cluster group is used.
   #
   # targets: ...
+  #
+  # Drift correction removes any external change made to resources managed by Fleet. It performs a helm rollback, which uses
+  # a three-way merge strategy by default. It will try to update all resources by doing a PUT request if force is enabled.
+  # Failed rollback will be removed from the helm history unless keepFailHistory is set to true.
```

Comment:
Do we want to add more information about _why_ forcing a rollback may be needed in some cases (depending on resource type, fields which may have been changed externally, etc)? Or would we leave that to the user, ie encouraging them to read about how `helm rollback` operates?

Reply:
done in https://github.com/rancher/fleet-docs/pull/75/commits/8c970d9a6d8528fb31b04db7a33798360e849295

Reply:
Thanks :)